pyqtgraph/parametertree/parameterTypes/basetypes.py

```diff
@@ -319,24 +325,26 @@ def __init__(self, param, depth):
         self.optsChanged(self.param, self.param.opts)
 
     def updateDepth(self, depth):
-        ## Change item's appearance based on its depth in the tree
-        ## This allows highest-level groups to be displayed more prominently.
-        if depth == 0:
-            for c in [0,1]:
-                self.setBackground(c, QtGui.QBrush(QtGui.QColor(100,100,100)))
-                self.setForeground(c, QtGui.QBrush(QtGui.QColor(220,220,255)))
-                font = self.font(c)
-                font.setBold(True)
-                font.setPointSize(font.pointSize()+1)
-                self.setFont(c, font)
-        else:
-            for c in [0,1]:
-                self.setBackground(c, QtGui.QBrush(QtGui.QColor(220,220,220)))
-                self.setForeground(c, QtGui.QBrush(QtGui.QColor(50,50,50)))
-                font = self.font(c)
-                font.setBold(True)
-                #font.setPointSize(font.pointSize()+1)
-                self.setFont(c, font)
+        """
+        Change set the item font to bold and increase the font size on outermost groups.
+        """
+        app = mkQApp()
+        palette = app.palette()
+        color = palette.base().color()
+        h, s, l, a = color.getHslF()
+        lightness = 0.5 + (l - 0.5) * .8
+        background = QtGui.QColor.fromHslF(h, s, lightness, a)
+        altBackground = color
```

Comment:
No problem, but you might be able to help future devs to figure this out faster.
What you now call "color" is the default background color.
What you call "background" is a modified background color.

Maybe swapping the names around makes that easier to follow (and saves a line):
``` python
background = palette.base().color()
h, s, l, a = background.getHslF()
lightness = 0.5 + (l - 0.5) * .8
altBackground = QtGui.QColor.fromHslF(h, s, lightness, a)
```

Reply:
I guess I did it this way because I didn't want to call `setBackground(altBackground)`, pick between using `setBackground` more than once, or writing an else.

Reply:
I think I'd be more confused seeing `setBackground(altBackground)` and having to trace my way through the code to figure out why it wasn't called with `background`